### ISSUE TITLE
Alphabetically list modules in translations

### ISSUE THREAD

**Issue by mrabsinthe** [PrestaShop/PrestaShop#13888]
International / Translations / Installed modules translations
It would be nice to have modules listed alphabetically by name in drop down.

**Comment by khouloudbelguith** [comment]
Hi @mrabsinthe,

Thanks for your report.
I think it could be a good idea, to have the list of modules are sorted by name in drop down.
![image](https://user-images.githubusercontent.com/16067358/58079474-fb28fb80-7ba8-11e9-9eca-8282c9916b2f.png)
@marionf, @colinegin what do you think?
Thanks!

**Comment by colinegin** [comment]
Yes, I agree, it seems like the obvious way of sorting ! @khouloudbelguith do you know how it is currently sorted ?

**Comment by khouloudbelguith** [comment]
@colinegin, I think that they are randomly sorted.
Thanks!

**Comment by Hlavtox** [comment]
Already implemented
